app/models/user.rb

```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
+  has_many :comments
   has_many :posts, foreign_key: :author_id
   has_many :likes
```

Comment:
Nice job making the associations between the user and other entities :+1: However, for these associations to fit the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) given, you should specify that the foreign key for the comments and likes is also `author_id`: 
```
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```
You should also make adjustments to your schema, since the foreign key in the likes table currently is `user_id` when it should be `author_id`.